OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -74,6 +74,11 @@
     private static final String COURSE_DATES_ENABLED = "COURSE_DATES_ENABLED";
     private static final String WHATS_NEW_ENABLED = "WHATS_NEW_ENABLED";
     private static final String COURSE_VIDEOS_ENABLED = "COURSE_VIDEOS_ENABLED";
+    /**
+     * On the time when deprecated activities {@link org.edx.mobile.view.MyCoursesListActivity},
+     * {@link org.edx.mobile.view.CourseDashboardActivity} and {@link org.edx.mobile.view.CourseOutlineActivity}
+     * will be removed, this flag will no longer be of any use.
+     */
     private static final String TAB_LAYOUTS_ENABLED = "TAB_LAYOUTS_ENABLED";
```

Comment:
`@Deprecated` can be added the top of this line.

Reply:
`At the time` instead of `On the time`.

Reply:
I will not prefer to add `@Deprecated` because there is no alternate available of it and developer suppose to use this flag in v2.13. Strike through on it will not make any sense.

Reply:
ok, fair enough.